packages/postcss-reduce-idents/src/lib/grid-template.js

```diff
@@ -53,6 +55,19 @@ module.exports = function () {
               }
             });
           }
+          if (child.type === 'word') {
+            const word = child.value;
+            if (word.startsWith('[') && word.endsWith(']')) {
```

Comment:
Maybe there's a better way to do it, but postcss-value-parser does not understand list of elements in brackets at all.